experiments/optimal-core-allocation/README.md

```diff
@@ -60,28 +59,28 @@ core.cpu() # if run with USE snap collector
 
 #### How to run in distributed mode with custom configuration file
 
-This example shows how to run the experiment using configuration file.
-All experiment flags can be provided using configuration file. Command line flag `-foo_bar` is equal to `FOO_BAR` option in configuration file.
+This example shows how to run the experiment using a configuration file.
+All experiment flags can be provided using a configuration file. Command line flag `-foo_bar` is equal to `FOO_BAR` option in configuration file.
 
 1. Generate default configuration: 
 
 ```bash
 sudo optimal-core-allocation -config-dump >example-configuration.txt
 ```
 
-2. Modify configuration to meet your needs:
+2. Modify configuration to meet your requirements:
 
 ```bash
 $EDITOR example-configuration.txt
 ```
 
-3. Run experiment with provided configuration:
+3. Run experiment with your configuration:
 
 ```bash
-sudo optimal-core-allocation -config example-configuration.txt -experiment_peak_load=800000
+sudo optimal-core-allocation -config example-configuration.txt
 ```
 
-4. (Optionally) Run experiment with override configuration flags.
+4. Run the experiment overriding configuration file values using flags:
```

Comment:
This could be marked as (optional) as an example that you actually *can* override things. What would you say?